src/backend/common/data/commands/tournament_placements.py

```diff
@@ -101,3 +103,137 @@ async def handle(self, db_wrapper, s3_wrapper):
+
+            # Team placements
+            async with db.execute("""
+                SELECT t.id as "tournament_id", t.name as "tournament_name", t.game, t.mode, tsp.squad_id, teams.name as "squad_name", teams.id as "team_id", t.date_start, t.date_end, tsp.placement, tsp.placement_description, tsp.is_disqualified
+                FROM tournament_players as tp 
+                INNER JOIN tournaments as t
+                ON tp.tournament_id = t.id 
+                LEFT JOIN tournament_squad_placements as tsp 
+                ON tp.squad_id = tsp.squad_id
+                INNER JOIN team_squad_registrations as tsr
+                ON tp.squad_id = tsr.squad_id
+                INNER JOIN team_rosters as tr ON
+                tsr.roster_id = tr.id
+                INNER JOIN teams
+                ON tr.team_id = teams.id
+                WHERE t.show_on_profiles = 1
+                AND tp.squad_id IS NOT NULL
+                AND tsp.squad_id IN (SELECT squad_id FROM team_squad_registrations)
+                AND tp.player_id = ?
+                ORDER BY t.date_start DESC;
+                """, (self.player_id,)) as cursor:
+                rows = await cursor.fetchall()
+                for row in rows:
+                    tournament_id, tournament_name, game, mode, squad_id, squad_name, team_id, date_start, date_end, placement, placement_description, is_disqualified = row
+                    tournament_team_results.append(PlayerTournamentPlacement(tournament_id, tournament_name, game, mode, squad_id, squad_name, team_id, date_start, date_end, placement, placement_description, is_disqualified, []))
+                results = PlayerTournamentResults(tournament_solo_and_squad_results, tournament_team_results)
+                return results
+
+
+@dataclass
+class GetTeamTournamentPlacementsCommand(Command[TeamTournamentResults]):
+    """
+    Get all tournament placement data for a particular team
+    """
+    team_id: int
+
+    async def handle(self, db_wrapper, s3_wrapper):
+        tournament_team_results = []
+        async with db_wrapper.connect(readonly=True) as db:
+            # Team placements
+            async with db.execute("""
+                SELECT t.id as "tournament_id", t.name as "tournament_name", t.game, t.mode, teams.id as "team_id", t.date_start, t.date_end, tsp.placement, tsp.placement_description, tsp.is_disqualified
+                FROM tournaments as t
+                LEFT JOIN tournament_squad_placements as tsp 
+                ON t.id = tsp.tournament_id
+                INNER JOIN team_squad_registrations as tsr
+                ON tsp.squad_id = tsr.squad_id
+                INNER JOIN team_rosters as tr ON
+                tsr.roster_id = tr.id
+                INNER JOIN teams
+                ON tr.team_id = teams.id
+                WHERE t.show_on_profiles = 1
+                AND teams.id = ?
+                ORDER BY t.date_start DESC;
```

Comment:
This query doesn't return results where the team doesn't have a placement since the inner join on team_squad_registrations depends on tournament_squad_placements, maybe have it depend on tournament ID instead and have tournament_squad_placements depend on the squad ID (so basically flipped)

Reply:
Makes sense - good call. I've made this change 👍
![image](https://github.com/user-attachments/assets/43a24fab-bccf-4e7f-8654-61142c39e7a6)